### ISSUE TITLE
sir, need help for my wrong result.

### ISSUE THREAD

**Issue by neuwangmeng** [zhixuhao/unet#8]
Thx for your git program, I start to learn FCN from your keras Unet. I am a student from NEU china.
I use your network to retrain on the given data, but I just get a 0.78 accuracy, it seems that the net can not learn anything after a fever epochs,the accuracy remains 0.78+ while loss can simultaneously be decreased , my batchsize=5,lr=0.001, and I just use 330 augmented pics. Why ? is it being incorrectly initialized?

**Comment by zhixuhao** [comment]
it seems that 330 images are not enough, so enlarge your training data and try again.

**Comment by neuwangmeng** [comment]
I try 1300+ pics, accuracy still remains 0.78+ after 10epochs, it`s wired. I am confused.

**Comment by zhixuhao** [comment]
maybe the initialization of weights is not very good, because the weights are initialized randomly, so try a few times more, and increase the step and batch size.

**Comment by skylight-hm** [comment]
@neuwangmeng I get the same result too.@zhixuhao Have you tested your code?

**Comment by neuwangmeng** [comment]
@skylight-hm  @你好，我也没有解决问题，之前我使用自己创建的数据集合出现的同样是上述问题，很奇怪。我也在东北高校学习，嘿嘿。

**Comment by neuwangmeng** [comment]
@zhixuhao anyway, thx for your code, I will try to finetune my weigts initialization later. Could you email me your model plz, my email address is wangmengbmie@outlook.com or 1163855905@qq.com, either is ok. thanks again.

**Comment by zhixuhao** [comment]
@neuwangmeng 
链接：http://pan.baidu.com/s/1qY3ARA8 

密码：2sy3

里面有数据集和模型权重

**Comment by YJonmo** [comment]
I ran the unet on my data (133 samples) with two epochs and after training, it gave blank results when the test images were applied. I know that 2 epochs are not enough for training but I think it should not give me the blank results.
